server/game/cards/02.2-DD/TallulahLulaMorgan.js

```diff
@@ -0,0 +1,30 @@
+const DudeCard = require('../../dudecard.js');
+
+class TallulahLulaMorgan extends DudeCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Tallulah "Lula" Morgan',
+            playType: ['noon'],
+            cost: [ 
+                ability.costs.bootSelf(),
+                ability.costs.payGhostRock(1)
+            ],
+            target: {
+                activePromptTitle: 'Choose a deed',
+                cardCondition: { 
+                    location: 'play area', 
+                    controller: 'current', 
+                    condition: card => card.owner !== this.controller 

```

Comment:
The new `equals` function should be used for cards and players from now on.

```suggestion
                    condition: card => !card.owner.equals(this.controller)
```